edison/config.py

```diff
@@ -0,0 +1,32 @@
+import secrets
+import importlib, inspect
+import sys, inspect
+
+config_dict = {}
+
+# If config_dict is empty this function builds it dynamically 
+# and returns the appropriate config object path.
+def get_config_object(env_keyword: str):
+    if(len(config_dict) == 0):
+        # Iterating through all config.py members
+        for name, obj in inspect.getmembers(sys.modules[__name__]):
+            # We're interested only with the derived classes of the Config class
+            if inspect.isclass(obj) and name != "Config":
```

Comment:
Do you care that it is a class inherited from Config, or any class as long as its not Config?
If you want to know if it inherits from Config you can use:
issubclass(obj, Config) 
Note this will return true for Config itself so you still need your code and name != "Config"

Reply:
Solved in PR #37 :)